apps/fio/tests/capacity/test.yml

```diff
@@ -0,0 +1,59 @@
+# TODO 
+# Change pod status checks to container status checks (containerStatuses)
+# O/P result
+
+- hosts: localhost
+  connection: local  
+
+  vars_files:
+    - test_vars.yml
+ 
+  tasks:
+   - block:
+       - name: Replace the pvc placeholder with test param in pvc spec
+         replace:
+           path: "{{ pvc_yaml_alias }}"
+           regexp: "testclaim"
+           replace: "{{ test_name }}"
+
+       - name: Replace the storageclass placeholder with provider
+         replace:
+           path: "{{ pvc_yaml_alias }}"
+           regexp: "testclass"
+           replace: "{{ lookup('env','PROVIDER_STORAGE_CLASS') }}"
+
+       - name: Create pvc 
+         shell: >
+           kubectl apply -f {{ pvc_yaml_alias }} -n litmus
+         args:
+           executable: /bin/bash
+
+       - name: Confirm pvc status is bound
+         shell: >
+           kubectl get pvc --no-headers -n litmus
+         args:
+           executable: /bin/bash
+         register: result
+         until: "'openebs-standard' and 'Bound' in result.stdout"
+         delay: 120
```

Comment:
120 seconds is a huge delay.. lets reduce this to 10 seconds and increase retries accordingly

Reply:
roger that